v2/tools/generator/internal/codegen/pipeline/resource_registration_file.go

```diff
@@ -428,22 +422,12 @@ func createGetResourceExtensionsImpl(codeGenerationContext *astmodel.CodeGenerat
 		Body: body,
 	}
 
-	f.AddReturn(astmodel.NewArrayType(astmodel.ResourceExtensionType).AsType(codeGenerationContext))
+	f.AddReturn(astmodel.NewArrayType(astmodel.ResourceExtensionType).AsType(context))
 	f.AddComments(funcComment)
 
-	return f.DefineFunc(), nil
-
+	return f.DefineFunc()
 }
 
-// createCreateSchemeFunc creates a createScheme() function like:
```

Comment:
why delete this comment?

Reply:
Thats my bad. Apologies!

Reply:
Might've been the result of merge from `main` I guess. I remember having a massive merge conflict in this file.